app/views/users/followers.html.erb

```diff
@@ -0,0 +1,6 @@
+<%=render partial: "users/navbar"%>
+<ul class="list-group list-group-flush">
+  <% @user.followers.each do |user| %>
```

Comment:
Great job on using partials to reduce repetitive code. For followers iterate over `follower` instead of user.